### PR TITLE
Project 2 Html forms

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
## Creating Html forms
In this project:
- I have rebuilt a form used by Mint.com for signing up new users

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <button class="footer_button">
+            <img src="./assets/icon-whitelock.png" alt="icon">
+            Create Account
+        </button>
+        <div class="footer">
+            By clicking Create account, you agree to our
+            <a href="#">Terms</a> and have read and acknowledge our <a href="#">Privacy Statement</a>.
+        </div>
+        <div class="lower_bottom">
+            <!-- <p>Invisible reCAPTCHA by Google Privacy Policy and Terms of Use.</p> -->
```
Comment (mosesogwo) [inline index.html]: Remove commented code

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @Emmanuelaaron :clap: Nice job. A few changes and you can :rocket:**

1. In your form, the `type` attribute for the phone field should be `tel`, not `number`.

2. Your form should be enclosed in `<form>` tags.

3. Package-lock.json is not required for this project. Remove it from the project folder.

4. Remove commented code from your html. (See my comment on index.html)

5. Please, set your footer in order. It should be centralized and the button on the right shouldn't be ther.

image

**Why has this project been done in only two commits?**

_**Kindly make these changes and submit another code review request.**_

**Good luck:smile:**

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi!

Well done fixing the issues previously pointed out, the project meets all the requirements, but... the [HTML validation is not passing](https://validator.w3.org/nu/?doc=https%3A%2F%2Femmanuelaaron.github.io%2FHTML-forms%2F) 😥:

- [ ] Fix HTML validation error.

[OPTIONAL]: You can add a `for` attribute to the labels to associate them with their corresponding input,, take a look at the label [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).

Status: **Request changes**.

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team,  You have done a great job with this project. :clap:

**Pull request merged**